stdlib/REPL/src/REPL.jl

```diff
@@ -1732,6 +1732,15 @@ function banner(io::IO = stdout; short = false)
         end
     end
 
+    loaded_user_modules = filter(names(Main,imported=true)) do m
+        typeof(getfield(Main, m)) <: Module && m ∉ (:Base, :Core, :Main)
+    end
+    if isempty(loaded_user_modules)
+        loaded_modules_string = ""
+    else
+        loaded_modules_string = "Loaded packages: $(join(loaded_user_models, ", "))"
+    end
```

Comment:
What happens if I put OmniPackage's direct and indirect dependencies in my `startup.jl`? Do you still like the outcome?

Reply:
<img width="690" alt="Screenshot 2025-05-09 at 11 29 51 AM" src="https://github.com/user-attachments/assets/14815f97-3d95-4ebb-bece-e3839394fe30" />


Note that Pkg and REPL are in this list because they were loaded as user packages via `-ie "using REPL, Pkg"` rather than the typical `require_stdlib` way.